.github/workflows/build.yaml

```
@@ -45,3 +45,14 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  docs-build:
+    if: ${{ startsWith(github.ref, 'refs/heads/branch-') }}
+    needs: cpp-build
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/custom-job.yaml@branch-23.04
+    with:
+      build_type: branch
+      node_type: "gpu-latest-1"
```

```suggestion
      node_type: "cpu4"
```

The `rapids-cmake` docs don't require GPUs to build, so we can just build them on a `cpu4` machine to save resources.